docs/pages/userdocs/hdf5io.dox

```diff
@@ -0,0 +1,202 @@
+/**
+ * @page hdf5io HDF5 I/O
+ *
+ * \section hdf5io_chunking Chunking
```

Comment:
This looks good but we currently don't have a way for the user to specify chunk size. It's defined in the `NWBFile.cpp` file. It's on the issue ticket list but not sure if we want to include it in the docs yet?

Reply:
I think it is fine to include here since you are using chunking even if a user cannot configure this yet.